src/components/EButtonBackToEvent.vue

```diff
@@ -0,0 +1,60 @@
+<script setup lang="ts">
```

Comment:
Does it really needs a dedicated component, why not use EButton and control the class from outside? or add another "color" to the button, such as "idle"?

Can not read the inline SVG, it is an arrow?